src/classes/ADDR_GoogleGeoAPI_Validator.cls

```diff
@@ -333,4 +333,4 @@ public with sharing class ADDR_GoogleGeoAPI_Validator implements ADDR_IValidator
     public static GoogleGeoResponse parse(String json) {
         return (GoogleGeoResponse) System.JSON.deserialize(json, GoogleGeoResponse.class);
     }
-}
```

Comment:
ADDR_GoogleGeoAPI_Validator.cls still has a newline character at the end of the file